### DISCUSSION TITLE
Blocking

### DISCUSSION THREAD

**Discussion by StaSiii** [fedorovvl/tso_client#103] [Ideas]
Hey,
First, I have to say THANK YOU for your work, it is awesome.

I have a question, is there a possibility, to block switching zones, when some actions is going on?
Eg, when script is sending explorers or when script is searching for collectables, it is not good to switch zones, game is crashing
Is there an option to block switching?

Maybe with a switch, placed in option menu, "yes/no"

**Comment by fedorovvl** [comment]
why you switching zone when script working? =))

**Comment by StaSiii** [comment]
When there are ~300 explorers is sending, it is easy to forget, that script is working and to not to switch to adventure or friend island.
